runtime/src/bank.rs

```diff
@@ -4609,11 +4609,7 @@ impl Bank {
             .write()
             .unwrap()
             .extend(rent_debits.into_unordered_rewards_iter());
-        if self
-            .feature_set
-            .is_active(&feature_set::cap_accounts_data_len::id())
-            && total_collected.account_data_len_reclaimed > 0
-        {
```

Comment:
I _just_ made this change yesterday, now to undo it... https://github.com/solana-labs/solana/pull/22918

Reply:
If I understand correctly you are not checking the feature gate here because `update_accounts_data_len()` won't throw an error / change the outcome and you want the calculation to happen for statistical purposes, right?

Reply:
Correct!